Include/internal/pycore_code.h

```diff
@@ -42,6 +42,10 @@ typedef struct {
     uint16_t defaults_len;
 } _PyCallCache;
 
+typedef struct {
+    int mask;
+} _PyCompareCache;
```

Comment:
You only need a few bits, so this could easily fit into the `index` field of the `_PyAdaptiveEntry`